src/sync/file-operations.ts

```diff
@@ -72,7 +72,7 @@ export function fileCompatible(filename: string, networkId: string, dbVersion: n
   return filename.endsWith(compSyncfile);
 }
 
-export function getHighestDbVersions(directoryDir: string, dbFileName: string): number {
+export function getHighestDbVersion(directoryDir: string, dbFileName: string): number {
```

Comment:
I didn't even see that ;-)

Reply:
initially I was going to return an array then the caller would decide what to do with it. but might as well just return highest like the name of method says.